README.md

```diff
@@ -40,7 +40,14 @@ To run the validator, execute the following command:
 ```
 python3 validator.py -f [path_to_file]
 ```
-Ensure that the path leads to a yaml file
+Ensure that the path leads to a yaml file.
```

Comment:
The usage message makes it look like `-f` is optional, but it appears to be required.
`usage: validator.py [-h] [-f PATH] [-u]`

Reply:
It is not required. When we discussed this yesterday, I decided to change it. You should just be able to run `validator.py -u` now.

Reply:
But if I'm reading the usage right, it's saying everything is optional.  If you don't use `-u`, then `-f` is required, right?

Reply:
Right. How do I write that?

Reply:
Hmm, I think it's:
`usage: validator.py [-h] [-u -f PATH] | [-f PATH] ]`

So you either specify `-u` with `-f PATH`, or you just optionally specify `-f PATH`.

Reply:
Ah interesting, okay! I will fix that and see about adding the new rule.